Automattic/wp-calypso
# Plugins Browser: Add missing info in the plugin browser item.

#### Changes proposed in this Pull Request
Addition of the following information on the Plugins Browser Item:

- [x] Last updated
- [x] Rating
- [x] Active Installs
- [x] Untested version warning

#### Testing instructions
* Go to `/plugins` page
* Confirm each plugin card has last updated, rating, and active install informations. Ex:

![Screen Shot 2021-12-06 at 14 58 37](https://user-images.githubusercontent.com/5039531/144906993-4d5e288b-eadd-485f-94a1-4d0f24058ada.png)

* Go to `/plugins` page
* Search for a term where will be showed plugins with untested versions. ex: application
* Check if the warning is being properly showed

![Screen Shot 2021-12-08 at 11 20 45](https://user-images.githubusercontent.com/5039531/145234933-b5e3de94-a6a9-4058-a021-bacd15409c63.png)


---

Fixes  #58319
